nixos/hosts/nixos/configuration.local.nix

```diff
@@ -0,0 +1,15 @@
+# Intentionally renamed from configuration.nix, keeping original generated config would be useful for debugging
+
+{
+  # config,
+  # lib,
+  # nixpkgs,
+  generic,
+  ...
+}:
+{
+  imports = [
+    ./hardware-configuration.nix
+    generic.nixosConfigurations.nixos.default
```

Comment:
Cannot share config directly, I should modulize for these pattern https://determinate.systems/posts/extending-nixos-configurations/